changelog.d/3909.misc

```diff
@@ -0,0 +1 @@
+Fix log format error in matrixfederationclient.py
```

Comment:
would be nice if this had the same text as #3906 so that the changelog entries get combined. I'll just change that on develop.